pytest_django/fixtures.py

```diff
@@ -129,6 +132,16 @@ def handle_noargs(self, **options):
             patch_for_test_db_setup()
 
 
+def _disable_native_migrations():
+    from django import get_version
+
+    if get_version() > '1.7':
```

Comment:
Shouldn't this be `get_version() >= '1.7'`? 1.7.1 will still be greater than 1.7, and I guess few people run 1.7, but still :)